lua/starfall/libs_sv/vehicles.lua

```diff
@@ -0,0 +1,25 @@
+assert( SF.Vehicles )
+
+local vehicle_methods = SF.Vehicles.Methods
+local vehicle_metamethods = SF.Vehicles.Metatable
+
+local wrap = SF.Vehicles.Wrap
+local unwrap = SF.Vehicles.Unwrap
+
+--- Gets the entity that is driving the vehicle
+-- @return The player driving the vehicle or nil if there is no driver
+function vehicle_methods:getDriver ()
+	SF.CheckType( self, vehicle_metamethods )
```

Comment:
As mentioned before, we don't really need these? Shouldn't be able to call this vehicle_method unless the type has the vehicle_metamethods underlying table, otherwise it then wouldn't be that specific type? Can somebody less tired/sleep deprived confirm these ramblings?